waltz-test/src/main/python/waltz_ducktape/tests/validation/scalability_test.py

```diff
@@ -97,3 +109,66 @@ def scale_up_replica(self, src_node_idx, added_node_idx, num_active_partitions,
         assert added_node_max_transaction_id == expected_max_transaction_id, \
             "New transactions failed to reach new replica, expected max transaction ID = {}, actual max transaction ID = {}" \
             .format(expected_max_transaction_id, added_node_max_transaction_id)
+
+    def get_number_of_partitions_assigned_to_server(self, server_cli, server):
```

Comment:
Would prefer moving this method to produce_consume_validate.py file to be in sync with other methods defined in that file.